lms/djangoapps/instructor/tests/test_email.py

```diff
@@ -31,7 +31,7 @@ def setUpClass(cls):
         # URL for instructor dash
         cls.url = reverse('instructor_dashboard', kwargs={'course_id': cls.course.id.to_deprecated_string()})
         # URL for email view
-        cls.email_link = '<button type="button" class="btn-link" data-section="send_email">Email</button>'
+        cls.email_link = '<button type="button" class="btn-link send_email" data-section="send_email">Email</button>'
```

Comment:
I am reusing the name from the data-section.  I can change this, but I am not sure what the secondary effects will be.